test/envs/regional-dr.yaml

```diff
@@ -55,6 +55,7 @@ templates:
       - addons:
           - name: submariner
             args: ["hub", "dr1", "dr2"]
+          - name: argocd
```

Comment:
You probably need to add:

    args: ["hub", "dr1", "dr2"]

Reply:
done